doc/leak_tracking/MOTIVATION.md

```diff
@@ -0,0 +1,81 @@
+
+Coming soon! See https://github.com/flutter/devtools/issues/3951.
```

Comment:
leak_tracker is not recommended for use yet, because of leaks in Flutter Framework.
We are piloting it. We will remove this line as soon as we have agreement to recommend the tool.